src/Cache.ts

```diff
@@ -7,9 +7,11 @@ import * as crypto from 'crypto';
 
 const d = debug('@electron/get:cache');
 
-const defaultCacheRoot = envPaths('electron', {
-  suffix: '',
-}).cache;
+const defaultCacheRoot =
+  process.env.ELECTRON_GET_CACHE_DIR ||
+  envPaths('electron', {
+    suffix: '',
+  }).cache;
```

Comment:
Here's my proposal: 
`mirrorVar` should be renamed to something like `userCustomValue`, it's not particularly specific to the mirror code _(except for the `MirrorOptions` part which can be genericized to `Record<string, unknown>`)_, and perhaps move it to a different file. Then we can change this to:
```suggestion
const defaultCacheRoot =
  userCustomValue(
    'cacheDir',
    {}, // this is set later on in the Cache constructor, I think, so passing a Record<string, unknown> isn't needed
    envPaths('electron', {
      suffix: '',
    }).cache
  );
```